src/applications/proxy-rewrite/proxy-rewrite-entry.jsx

```diff
@@ -231,18 +231,12 @@ const scriptPaths = [
 
 const linkPaths = ['/generated/styleConsolidated.css'];
 
-function removeInjectedHeaderFooter(
-  links = linkPaths,
-  scripts = scriptPaths,
-  docHead = document.head,
-) {
-  Array.from(docHead.getElementsByTagName('script'))
-    .filter(node => scripts.includes(node.getAttribute('src')))
```

Comment:
originally this was suppose to match the `src` attribute against anything in the `scripts` array but didn't quite write it correctly. conflated Array.includes and String.includes

Reply:
So this worked in testing because when running against localhost the strings matched correctly. In prod, though the src attributes include the S3 bucket as the hostname.